examples/caching/index.html

```diff
@@ -1,5 +1,5 @@
 <!DOCTYPE html>
-<html>
+<html class="theme-settings">
```

Comment:
We've never had one implemented in gaia-fast-list because of the cost added to first-paint. Now we have the `.complete` promise hook, it's probably safe to add it there.